AutomatedTesting/Gem/PythonTests/editor/EditorScripts/EntityOutliner_EntityOrdering.py

```diff
@@ -0,0 +1,145 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+
+class Tests:
+    entities_sorted = (
```

Comment:
nit: A better name might be entities_sorted_result or entities_sorted_status or entities_sorted_message

Reply:
I agree that this is a little conventionally funky, but I'm inclined to leave as-is for now because I think this is in line with the convention established by other tests, and I'm striving to be consistent with other tests, e.g.
```
class Tests:
    asset_filtered = (
        "Asset was filtered to in the Asset Browser",
        "Failed to filter to the expected asset"
    )
    asset_type_filtered = (
        "Expected asset type was filtered to in the Asset Browser",
        "Failed to filter to the expected asset type"
    )
```